kedro-telemetry/RELEASE.md

```diff
@@ -1,5 +1,31 @@
-# Upcoming Release 0.2.4
-* Change reference to `kedro.pipeline.Pipeline` object throughout test suite with `kedro.modular_pipeline.pipeline` factory.
+# Release 0.2.4
+* Added consent checking for collecting project statistics.
+
+The `kedro-telemetry` plugin asks for consent to collect command run data. However, when we introduced functionality to understand the size of projects by counting the number of datasets, pipelines and nodes, the plugin did not verify consent, leading to the plugin collecting data without user consent. This bug affects users with `kedro-telemetry >= 0.2.2` installed, 22.5% of our user base.
+
+<img width="730" alt="Screenshot 2023-04-04 at 16 17 43" src="https://user-images.githubusercontent.com/43755008/229840293-fea91403-b02d-4221-9167-71d3a6196cc5.png">
```

Comment:
Should this picture be on the repo somewhere? And is there any reason to use the HTML `<img>` rather than the markdown `![...](...)` syntax like we normally do for images?

Reply:
(I'd also be happy to just not have the graph there at all because we already say it's 22.5% of the user base above and I doubt any user really cares about seeing the full breakdown. But it is a nice and interesting graph)